pkg/server/config/config.go

```diff
@@ -444,6 +464,21 @@ func (cfg *Config) Verify() error {
 	return nil
 }
 
+func (cfg *Config) verifyAccessControl(experimental string) error {
+	if experimental == "enable-access-control" && cfg.AccessControl.Enabled && cfg.AccessControl.StoreID != "" && cfg.AccessControl.ModelID != "" {
+		_, err := ulid.Parse(cfg.AccessControl.StoreID)
+		if err != nil {
+			return fmt.Errorf("config 'access-control-store-id' must be a valid UUID")
```

Comment:
nit: 
```suggestion
			return fmt.Errorf("config 'access-control-store-id' must be a valid ULID")
```

UUID is a different format